canonical-web-and-design/vanilla-framework
# Remove showcase page

## Done

- Removed showcase page
- Removed links to it, and added a redirect (back to the homepage)

Fixes #3805 

## QA

- Try to visit https://vanilla-framework-3810.demos.haus/showcase
- See that you're taken back to the homepage

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
